manual/maas-sru-19.4.33.txt

```diff
@@ -0,0 +1,8323 @@
+
+DISCLAIMER: FAILURE here is KNOWN BUG that hasn't SRU'd yet for maas http://pad.lv/1845351
```

Comment:
I want to make sure I'm following; the failure is due to a known bug _in MAAS_, right?